src/Powercord/index.js

```diff
@@ -45,7 +45,7 @@ let coremods;
  */
 class Powercord extends Updatable {
   constructor () {
-    super(join(__dirname, '..', '..'), '', 'powercord');
+    super(join(__dirname, '..', '..'), '', 'plugin', 'powercord');
```

Comment:
I think type `plugin` isn't appropriate here, maybe `base`?
```suggestion
    super(join(__dirname, '..', '..'), '', 'base', 'powercord');
```